portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,18 +26,23 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
```

Comment:
Would it be better to use /comments URL?

Reply:
Sure, this can be done.

Reply:
For your info, which I commented for others.

Best practice of REST API resource name should be plural.
Further reference can be found here:
https://google.aip.dev/122#collection-identifiers